chart/k8gb/templates/external-dns/external-dns-azure-auth.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if and .Values.azuredns.enabled .Values.azuredns.createAuthSecret }}
```

Comment:
we probably should create this Secret with Credentials outside of the helm chart and point to it from the `values.yaml` for k8gb to pick up